lambdas/test/apis/profiles/controller/profiles.spec.ts

```diff
@@ -118,7 +118,7 @@ function profileWith(
         description: 'description',
         hasClaimedName: true,
         avatar: {
-          bodyShape: {},
+          bodyShape: 'bodyShape',
```

Comment:
From the change in `profiles.ts` I don't understand why this attribute changes from an empty object to the string 'bodyShape'.

Reply:
Because it was never an object, it was a string. The mock in the test was wrong